### PR TITLE
Add tasks

### PR DESCRIPTION
**Pull request by john-kheir**

### CODE REVIEW

File: functional_testing/Openvcloud/ovc_master_hosted/OVC/c_advanced/date_integrity.py [hunk 1]
```diff
@@ -1,65 +0,0 @@
...
-            machine_id = self.cloudapi_create_machine(cloudspace_id=cloudspace_id,
-                                                      size_id=size['id'],
-                                                      image_id=image['id'],
-                                                      disksize=disksize, stackid=sn_id)
-
-
-        # Get cloupdspace ip
-        self.lg("")
-
-        self.lg('%s ENDED' % self._testID)
```
Comment (ramezsaeed) [inline functional_testing/Openvcloud/ovc_master_hosted/OVC/c_advanced/date_integrity.py]: why did u removed this test ?

File: functional_testing/Openvcloud/ovc_master_hosted/OVC/a_basic/machine_tests.py [hunk 2]
```diff
@@ -721,3 +721,12 @@ def test011_windowsVM_with_different_sizes(self):
                 self.assertTrue(machineId)
 
         self.lg('%s ENDED' % self._testID)
+
+    def test012_delete_nonexisting_machine(self):
```
[the remark this answers is not in the document]
Comment (john-kheir) [reply functional_testing/Openvcloud/ovc_master_hosted/OVC/a_basic/machine_tests.py]: Done

File: functional_testing/Openvcloud/ovc_master_hosted/OVC/b_extended/cloudspace_tests.py [hunk 3]
```diff
@@ -73,4 +74,41 @@ def test001_validate_deleted_cloudspace_with_running_machines(self):
...
+
+    @unittest.skip('Not Implemented')
+    def test004_get_OpenVPN_config_file(self):
+        """ OVC-000
+        *Test case for getting OpenVPN config file for cloudspace.*
+
+        **Test Scenario:**
+        #. Create new cloudspace CS1.
+        #. Get OpenVPN config file for CS1, should succeed .. with /cloudapi/cloudspaces/getOpenvpnConfig API, should succeed.
+        #. Check that Config file is right.
```
[the remark this answers is not in the document]
Comment (john-kheir) [reply functional_testing/Openvcloud/ovc_master_hosted/OVC/b_extended/cloudspace_tests.py]: Done

File: functional_testing/Openvcloud/ovc_master_hosted/OVC/a_basic/users_tests.py [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+        """ OVC-000
+        * Test case for check user authentication and passsword update. *
+
+        **Test Scenario:**
+
+        #. Create user (U1) with admin access.
+        #. Authenticate U1 with POST /cloudapi/users/authenticate API,should return session key[user1_key] .
+        #. Use U1's key to list accounts for U1, should succeed.
+        #. Use U1's key to update U1's password, should succeed.
+        #. Check that user1's password has been reset successfully.
```
[the remark this answers is not in the document]
Comment (john-kheir) [reply functional_testing/Openvcloud/ovc_master_hosted/OVC/a_basic/users_tests.py]: done

**Pull request merged**